packages/models-library/src/models_library/functions_catalog/constants.py

```diff
@@ -0,0 +1,5 @@
+from typing import Final
+
+FUNCTION_SERVICE_KEY_PREFIX: Final[str] = "simcore/services/frontend"
+# NOTE: due to legacy reasons, the name remains with 'frontend' in it but
+# it now refers to a more general group: function sections that contains front-end services as well
```

Comment:
Question: will this change in the future or will it stay like this?

Reply:
I do not know. For that reason I have created the variable, so that if it changes I only will refactor here